designs/2019-config-simplification/README.md

```diff
@@ -128,13 +287,13 @@ exports.config = {
 
 Here, it is the `ruledefs` that assigns the name `react` to the rules from `eslint-plugin-react`. The reference to `react/` in a rule will always look up that value in the `ruledefs` key.
 
-**Note:** If the config `extends` another config that already has a `ruledefs` namespace defined, then an error is thrown. In this case, if a config already has a `react` namespace, then attempting to combine with another config that has a `react` namespace will throw an error. This is to ensure the meaning of `namespace/rule` remains consistent.
+**Note:** If a config is merged with another config that already has a `ruledefs` namespace defined, then an error is thrown. In this case, if a config already has a `react` namespace, then attempting to combine with another config that has a `react` namespace will throw an error. This is to ensure the meaning of `namespace/rule` remains consistent.
```

Comment:
Should this say "the same" (or "a namespace with the same name as the config to be merged", or similar)? The way I'm currently reading the sentence, it could be interpreted as *any* two configs that have *anything* in ruledefs will error, but I think it's supposed to be if they export ruledefs under the same namespace name.

Reply:
Yes, that would be a much clearer explanation of my intent here. Thanks, I'll update that.